scripts/rpm/fapolicy-analyzer.spec.j2

```diff
@@ -10,15 +10,16 @@ URL:            https://github.com/ctc-oss/fapolicy-analyzer
 Source0: %{url}/archive/%{git_commit}/%{name}-%{git_commit}.tar.gz
 
 BuildArch:      x86_64
-BuildRequires:  python3 >= 3.9, python3-devel, python3-pip, python-wheel, dbus-devel, rust-packaging
-{% if (fedora_ver | int) >= 34 %}BuildRequires:  python-setuptools-rust{% endif %}
-Requires:       python3 >= 3.9, gtk3, python3-gobject, python3-events, python3-configargparse
-Requires:       dbus-libs
+Requires:       {{ pyrpm }} >= 3.9
+BuildRequires:  {{ pyrpm }} >= 3.9, {{ pyrpm }}-rpm-macros, {{ pyrpm }}-devel, {{ pyrpm }}-pip, {{ pyrpm }}-wheel
+BuildRequires:  dbus-devel, cargo
+Requires:       gtk3, dbus-libs
```

Comment:
Is it possible to combine the Requires: lines together, and the BuildRequires: lines together?